indico/modules/events/abstracts/operations.py

```diff
@@ -91,8 +92,11 @@ def delete_abstract_files(abstract, files):
                        data={'Files': ', '.join(f.filename for f in files)})
 
 
-def create_abstract(event, abstract_data, custom_fields_data=None, send_notifications=False):
-    abstract = Abstract(event=event, submitter=session.user)
+def create_abstract(event, abstract_data, custom_fields_data=None, send_notifications=False, submitter=None):
+    abstract = Abstract(event=event, submitter=session.user if submitter is None else submitter)
+    if submitter:
```

Comment:
That logic is a bit weird - having a submitter implying that it's an invited abstract is not that obvious.

`invited=True` kwarg instead?